src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -517,6 +519,7 @@ void ResumptionDataTest::SetSubmenues() {
   sm::SmartObject sm_comm;
   for (uint32_t i = 10; i < kCountOfSubmenues_ + 10; ++i) {
     char numb[12];
+    // cppcheck-suppress invalidPrintfArgType_sint
```

Comment:
@Ypostolov Do not suppress this issue. You can find solution at several comment above.